Dockerfile

```
@@ -4,8 +4,10 @@ RUN apt-get update && apt-get -uy upgrade
 RUN apt-get -y install ca-certificates && update-ca-certificates
 
 FROM scratch
-
+# Allow Glog to log to disk
+COPY --from=0 /tmp /tmp
```

Are the contents of /tmp really needed? This feels like it could be simplified.

```suggestion
RUN mkdir -p /tmp
```

---

Sorry was out of context here. As we are using scratch `mkdir` isn't available and copying it over from=0 is the working solution. Still not sure I agree with that completely as it might copy over more than just the empty directory.

How about making an empty directory in stage 0 and then copying over that one in the final stage.

---

Yeah, that is probably the right solution, good catch, I'll fix it to that.